core/text.py

```diff
@@ -43,6 +43,14 @@ def format_item(self, item, ql=None, with_icon=True):
 
         return result
 
+    def generate_item(self, item, ql, synonym=None):
+        if synonym:
+            return {"icon_%s" % synonym: self.make_item(item.lowid, item.highid, ql, self.make_image(item.icon)),
+                    "text_%s" % synonym: self.make_item(item.lowid, item.highid, ql, item.name)}
+        else:
+            return {"icon" % synonym: self.make_item(item.lowid, item.highid, ql, self.make_image(item.icon)),
+                    "text" % synonym: self.make_item(item.lowid, item.highid, ql, item.name)}
+
```

Comment:
`"icon" % synonym` should just be `"icon"` and `"text" % synonym` should just be `"text"` in the else clause, correct?